crates/pixi_build_types/src/project_model.rs

```diff
@@ -296,6 +305,10 @@ pub struct BinaryPackageSpecV1 {
     /// The sha256 hash of the package
     #[serde_as(as = "Option<SerializableHash<Sha256>>")]
     pub sha256: Option<Sha256Hash>,
+    /// The URL of the package, if it is available
+    pub url: Option<Url>,
+    /// The license of the package
+    pub license: Option<String>,
```

Comment:
What is the reason to include `url` and `license` here?

Reply:
These are also fields of (`Nameless`)`MatchSpec` and otherwise couldnt be represented in the pixi representation.